homeassistant/components/zhong_hong/strings.json

```
@@ -0,0 +1,43 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Set up ZhongHong HVAC",
+        "description": "Configure connection to your ZhongHong HVAC gateway.\n\nDefault port is {default_port} and default gateway address is {default_gateway}.",
+        "data": {
+          "host": "Gateway IP Address",
```

Home Assistant requires sentence-casing for all strings, including headers.

```suggestion
          "host": "Gateway IP address",
```